daemon/algod/api/server/v2/dryrun.go

```diff
@@ -400,7 +400,8 @@ func doDryrunRequest(dr *DryrunRequest, response *generated.DryrunResponse) {
 			ep.Debugger = &debug
 			pass, err := logic.Eval(stxn.Lsig.Logic, ep)
 			var messages []string
-			result.Disassembly = debug.lines
+			result.Disassembly = debug.lines          // Keep backwards compat
+			result.LogicSigDisassembly = &debug.lines // Also add to Lsig specific
```

Comment:
Why does these have different types?  They look the same in oas3.yml and oas2.json

Reply:
Since dry-run requests have at least one program (lsig or app) the disassembly will always get populated. But lsig-disassembly would not.

Though I think dry run requests can have some normal txns in them (just a simple pay), right? That would not populate disassembly at all.  You you get DryrunTxnResponse for such?

Reply:
yea, I get an object with `{disassembly:""}`